Source/Csla/DataPortalClient/HttpCompressionProxy.cs

```diff
@@ -62,27 +62,32 @@ protected override void SetHttpRequestHeaders(HttpRequestMessage request)
     /// </summary>
     protected override WebClient GetWebClient()
     {
-      var client = new CompressionWebClient(Timeout);
```

Comment:
Why did your remove the `!`? The framework guarantees it's not null. Furthermore with this change a lot of null checks are now missing.

Reply:
In this case we because we derive from our own class here.

Reply:
In one class there was the` ! `and in the other wasn't, I just made them similar, I can put it back.

In fact I don't like the `! ` because we are forcing something that the compiler isn't sure about, I like better to enable nullable and mark all nullable as errors.

and evn after that I just use ! with a good justification

Reply:
Looks like I missed the other part 😅. That project should already be NRT aware and yes I'll make a PR to enable NRT as errors for already migrated projects. So please revert that change and I'll revisit this part in the near future.

Reply:
#4171 changes the warnings to be errors